lib/post_widget.dart

```diff
@@ -0,0 +1,133 @@
+import 'package:flutter/material.dart';
+
+class PostWidget extends StatelessWidget {
```

Comment:
Đổi tên file thành post_list_widget.dart luôn nhé
```suggestion
class PostListWidget extends StatelessWidget {
```